api/routes/users.js

```diff
@@ -4,20 +4,20 @@ const db = require('../db');
 const dbUtils = require('../utils/db.js');
 const apiUtils = require('../utils/api.js');
 const jwt = require('jsonwebtoken');
-const secret = "secret"
+const secret = "FsT8VApMlKjmCgUPKIrv"
```

Comment:
I think it'd be awesome if the secret was saved in an environment variable instead of being hardcoded

Reply:
fixed